src/test/regress/sql/multi_explain.sql

```diff
@@ -17,11 +41,21 @@ EXPLAIN (COSTS FALSE, FORMAT JSON)
 	SELECT l_quantity, count(*) count_quantity FROM lineitem
 	GROUP BY l_quantity ORDER BY count_quantity, l_quantity;
 
+-- Validate JSON format
```

Comment:
I don't think if it is required or not, but, I also tested with subquery pushdown it looks good.